opslevel/datasource_opslevel_team.go

```diff
@@ -33,6 +33,45 @@ type TeamDataSourceModel struct {
 	ParentId    types.String      `tfsdk:"parent_id"`
 }
 
+var teamDatasourceSchemaAttrs = map[string]schema.Attribute{
```

Comment:
Shouldn't this only have `"alias"`, `"id"`, and `"name"` attributes? With the rest passed into `teamAttributes()` below?

Looking at the [old team datasource](https://registry.terraform.io/providers/OpsLevel/opslevel/0.11.0/docs/data-sources/team) and [old teams datasource](https://registry.terraform.io/providers/OpsLevel/opslevel/0.11.0/docs/data-sources/teams) these are the only fields that apply to both datasource types